backend/de.metas.business.rest-api-impl/src/main/java/de/metas/rest_api/v2/order/sales/SalesOrderRestController.java

```diff
@@ -229,7 +229,7 @@ private JsonSalesOrderAttachment toSalesOrderAttachment(final int salesOrderId,
 		return JsonSalesOrderAttachment.builder()
 				.salesOrderId(String.valueOf(salesOrderId))
 				.id(AttachmentEntryId.getRepoId(entry.getId()))
-				.type(JsonAttachmentType.valueOf(entry.getType().toString()))
+				.type(JsonAttachmentSourceType.valueOf(entry.getType().name()))
```

Comment:
instead please add a little method with a switch, to make sure that renamings in one of the two enums won't break it
(also from de.metas.rest_api.v1.order.sales.SalesOrderRestController and de.metas.rest_api.v1.ordercandidates.impl.OrderCandidatesRestController)